tests/BaseTest.py

```diff
@@ -0,0 +1,22 @@
+import unittest
+import json
+import pprint
+from app import create_app
+
+
+class BaseTest (unittest.TestCase):
+
+    def setUp(self):
+        self.app = create_app("testing")
+        self.client = self.app.test_client
+
+        self.requests = {
+            "requests": "This is a request",
+            "type": "repair"
+        }
+    def tearDown(self):
```

Comment:
Add a new line between the `tearDown` and the `setUp` functions. you may need to use a linter `pylint` or `flake8` to iron out all errors that are in your code